components/FormikFields/PaperInputPicker/index.js

```diff
@@ -1,9 +1,8 @@
 import * as React from 'react';
 import {
-  View,
-  Text
+  View, Text
 } from 'react-native';
-import { TextInput, Button } from 'react-native-paper';
+import { TextInput, Button, Title } from 'react-native-paper';
```

Comment:
This was a hotfix to add labels for select forms (really should be in a different PR)